python/src/aiconfig/editor/server/server.py

```diff
@@ -167,26 +167,27 @@ async def run() -> FlaskResponse:
     aiconfig = state.aiconfig
     request_json = request.get_json()
 
-    _op = make_op_run_method(MethodName("run"))
-
-    def _get_op_args():
-        prompt_name = request_json.get("prompt_name", None)
-        stream = request_json.get("stream", True)
-        LOGGER.info(f"Running prompt: {prompt_name}, {stream=}")
-        inference_options = InferenceOptions(stream=stream)
-        return Ok(
-            OpArgs(
-                {
-                    "prompt_name": prompt_name,
-                    #
-                    "options": inference_options,
-                }
-            )
-        )
-
-    op_args = _get_op_args()
-
-    return run_aiconfig_operation_with_op_args(aiconfig, "run", _op, op_args)
+    try:
+        prompt_name = request_json["prompt_name"]
+        params = request_json.get("params", {})
+        stream = request_json.get("stream", False)
+        options = InferenceOptions(stream=stream)
+        kwargs = request_json.get("kwargs", {})
```

Comment:
kwargs is really specific to python and not something any client should ever need to send up -- the endpoint request should always have a clear type of what's accepted instead of being open

Reply:
whatever kwargs is for, I can't think of a better way to allow the frontend to control it. We would have to hard-code specific kwargs that can be accepted. Thoughts @Ankush-lastmile ?

Reply:
I'm cool with hard-coding. If it's from the client, they should be explicit and conform to our API anyways

Reply:
If we're hard-coding, can we change the signature of `run()` to define these hard-coded things instead? That would fix a lot of quiet problems as a nice side effect

Reply:
I think we can just remove it here.

 I think it can be removed from the config api itself, as well. Looking at the python code for config, abstract model_parser run method doesn't support kwargs and neither should individual parser implementations. It seems to be used for `run_with_dependencies` but I think that should be part of InferenceOptions instead

Reply:
Agreed, run_with_dependencies should be in InferenceOptions. Although, funnily enough, InferenceOptions has its own kwargs so that doesn't really solve the problem. Anyway I'll remove it from the endpoint and we'll just have to hard-code or be satisfied with not supporting extra options for now.

Reply:
https://github.com/lastmile-ai/aiconfig/issues/664

Reply:
Ok cool, added an issue to track this here: https://github.com/lastmile-ai/aiconfig/issues/663. It's a good bootcamp task, let's delete from here to get this PR unblocked and landed

Reply:
I see we both made an issue for it. I don't know if it's as simple as just deleting the kwargs since we need to move existing run_with_dependencies off of the kwargs usage

Reply:
^You're right, I closed my issue (https://github.com/lastmile-ai/aiconfig/issues/663) and commented on yours (https://github.com/lastmile-ai/aiconfig/issues/664) because yours was more descriptive, thanks for adding!

Reply:
What do you mean move existing run_with_dependencies off kwargs usage? I *think* adding the argument and removing kwargs should work except at call sites that give a keyword arg other than run_with_dependencies.